src/mono/msbuild/android/build/AndroidApp.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <PropertyGroup>
+    <RuntimeIdentifier>$(TargetOS.ToLowerInvariant())-$(TargetArchitecture.ToLowerInvariant())</RuntimeIdentifier>
+    <UseMonoRuntime>true</UseMonoRuntime>
+    <UseMonoJustInterp Condition="'$(RunAOTCompilation)' == 'true' and '$(MonoForceInterpreter)' == 'true'">true</UseMonoJustInterp>
+
+    <AndroidBuildAppAfterThisTarget Condition="'$(AndroidBuildAppAfterThisTarget)' == ''">Publish</AndroidBuildAppAfterThisTarget>
```

Comment:
It doesn't seem like we had constrained the execution timing of `AndroidBuildApp` based on Publish before this PR, does it to need it now?

Reply:
I may be misunderstanding your question, but the tests have always executed after publish.  This property allows us to potentially change the target we run after.  I don't anticipate it changing any time soon, but running after build could be a possibility.

Reply:
The build order has that targets listed in `AfterTargets` are ran after the current target is executed https://docs.microsoft.com/en-us/visualstudio/msbuild/target-build-order?view=vs-2022

If we want to change the target that the current target runs after, I believe we would want to set the BeforeTargets instead.

Reply:
So in this case, `AndroidBuildApp` would be executed before `Publish`, which seems problematic because we would want to have the publishdir before trying to AOT/compile.

Reply:
My mistake, I got it backwards!